db/seeds.rb

```diff
@@ -1,7 +1,16 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+video = Video.create!(
+  name: "foo",
+  description: "description Praesent venenatis justo ut venenatis consectetur.
```

Comment:
Run away from Lorem Ipsum! This shit does not resemble anything real ever.

Always use realistic texts (if you can't find real examples, make them up).